Process.h

```diff
@@ -19,6 +19,7 @@ in the source distribution for its full text.
 
 #define PROCESS_FLAG_IO              0x00000001
 #define PROCESS_FLAG_CWD             0x00000002
+#define PROCESS_FLAG_TTY             0x00000004
```

Comment:
I don't quite like this flag being all-platform visible, when only Darwin does anything useful with it. Why not create this flag in `darwin/DarwinProcess.h` similar to what I referenced with those flags for Linux.

Reply:
Good catch and fixed, I didn't realize that there were OS specific flags.